### DISCUSSION TITLE
Do not hide/show virtual keyboard when connected to a physical keyboard

### DISCUSSION THREAD

**Discussion by macjl** [isontheline/pro.webssh.net#929] [Ideas]
Hello,

I have some issues when using WebSSH with an iPad connected to physical keyboard. When I click on the screen, I loose focus, and toolbar disappears. I think the app wants to hide the virtual keyboard, that’s why I can’t enter commands anymore. I have to click again on screen, which displays the toolbar again, and then I can type commands.

Maybe you should disable the mechanism that hide/show the virtual keyboard when physical keyboard is connected ?

**Comment by isontheline** [comment]
Bonjour Jean-Laurent 🇫🇷 👋

You're right keyboard management could be improved when a hardware one is connected.

I've just opened an issue with your request : https://github.com/isontheline/pro.webssh.net/issues/930

I can't tell you when I will be able to work on it but I will.

Belle journée à vous ☀️

**Reply by isontheline** [reply]
@macjl 
The next upcoming beta release will let you to set if you want to hide keyboard on :
* screen touch
* button press
* both



![Image](https://github.com/isontheline/pro.webssh.net/assets/44212923/f387b0e8-b594-45f3-98e6-7a31a7757308)

Should be available in a few hours : https://testflight.apple.com/join/QSrBK59z
